jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   collectCoverage: true,
-  collectCoverageFrom: ['src/**/*.js', '!src/**/stories/*'],
+  collectCoverageFrom: ['src/**/*.{js,jsx}', '!**/node_modules/**'],
```

Comment:
@leSamo, is it safe to exclude `'src/**/stories/*'`-like paths from coverage collection?